source/sink-connector/configuration-properties/mongodb-namespace.txt

```diff
@@ -72,16 +72,17 @@ Settings
        | **Accepted Values**: A MongoDB database name
 
    * - **collection**
-     - | *Required*
-       |
-       | **Type:** string
+     - | **Type:** string
        |
        | **Description:**
        | The name of the MongoDB collection to which the sink connector
          writes. If your sink connector follows multiple topics, this
          is the default collection for any writes that are not otherwise
          specified.
-       |
+
+       | **Default**:
+       | The topic name.
```

Comment:
S: remove period and make this one line. Also, It seems like the convention for whether the colon is bolded or not is not established on this page. maybe out of scope but you could consider changing all of them to be one way or the other. Most other places, I have seen the colon un-bolded

```suggestion
       | **Default:** The topic name
```